packages/client/src/GlobalWindow.re

```diff
@@ -22,7 +22,7 @@ module Navigator = {
         let coords = coordinatesFromJs(positionAsRecord.coords);
         cb(coords);
       },
-      Js.log,
+      Js.log2("Navigator.GeoLocation"),
```

Comment:
Need?

Reply:
The global geolocation takes an error-callback at that argument-position, we could do `ignore()` I guess, but might be handy for debugging for us or actual users!